src/components/Trends/TrendsPanel.jsx

```diff
@@ -59,6 +78,9 @@ export class TrendsPanel extends React.Component {
   }
 
   _phaseMap() {
+    const minDate = moment(this.props.date_received_min).format('MM/DD/YYYY')
```

Comment:
I would move these down to the mapStatetoProps so you can just do this.props.minDate